src/services/orderbook_service.ts

```diff
@@ -69,10 +69,18 @@ export class OrderBookService {
     ): Promise<OrderbookResponse> {
         const [bidSignedOrderEntities, askSignedOrderEntities] = await Promise.all([
             this._connection.manager.find(SignedOrderEntity, {
-                where: { takerAssetData: baseAssetData, makerAssetData: quoteAssetData },
+                where: {
+                    takerAssetData: baseAssetData,
+                    makerAssetData: quoteAssetData,
+                    expirationTimeSeconds: MoreThan(new Date().getTime()),
```

Comment:
I believe this `new Date().getTime()` is milliseconds whereas `expirationTimeSeconds` is in seconds.
```suggestion
                    expirationTimeSeconds: MoreThan(Math.floor(new Date().getTime() / 1000)),
```